### DISCUSSION TITLE
Can a photo be associated to multiple persons (a photo with multiple person in it) ?

### DISCUSSION THREAD

**Discussion by dgedgedge** [nextcloud/recognize#1021] [General]
When a photo contain multiple persons, it should be associated to multiple persons.
Is it the current behaviour ?
Maybe the current behaviour is to find only one person in each photo => the person to photo relation is a 1 to 1 relation.
I would have imagined that it should have been a 1 to many relation :
1 photo to many persons.

**Comment by marcelklehr** [comment]
> the person to photo relation is a 1 to 1 relation.

this is not the case.

**Comment by dgedgedge** [comment]
OK so its at least a 1 to N relation.
Thanks.
